airbyte-webapp/src/packages/cloud/views/auth/components/FormContent.tsx

```diff
@@ -8,8 +8,8 @@ import { Header } from "./Header";
 
 const MainBlock = styled.div`
   width: 100%;
-  height: calc(100% - 100px);
   display: flex;
+  flex: 1 0 auto;
```

Comment:
:information_source: This is the other part of making sure the primary area of the left side in the auth forms is actually scaling to all the place it can have, now using flexbox instead of some `height` calculation (that wasn't accurate).